src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,131 @@
+package duke.ui;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+
+/**
+ * Deals with interactions with the user
+ */
+public class Ui {
+    /**
+     * Sends greeting message to user
+     */
+    public void sendGreetingsMessage() {
+        System.out.println("Hello I'm duke.Duke\n" + "What can I do for you?");
+    }
+
+    /**
+     * Sends goodbye message to user
+     */
+    public void sendGoodByeMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Sends task details, formatted to user
+     * @param taskNumber
+     * @param task
+     */
+    public String sendTaskDetails(int taskNumber, Task task) {
+        return taskNumber + ". " + task + "\n";
+    }
+
+    /**
+     * Loop through task lists to print task details
+     * @param tasks
+     * @return task details
+     * @throws DukeException
+     */
+    public String getTaskListDetailsMessage(TaskList tasks) throws DukeException {
+        String s = "Here are the tasks in your list:\n";
+        try {
+            for (int i = 0; i < tasks.getSize(); i++) {
+                s = s + this.sendTaskDetails(i + 1, tasks.getTask(i));
+            }
+        } catch (Exception e) {
+            throw new DukeException(e.getMessage());
+        }
+        return s;
+    }
+    /**
+     * Return task details based on keyword by user
+     * @param tasks
+     * @return task details
+     * @throws DukeException
+     */
```

Comment:
Adding a space between the description and the parameters will better follow the coding standards. I found this issue in all of your Javadocs